ui/Screen/Project/Commit.svelte

```diff
@@ -44,6 +44,232 @@
 
       // TODO(cloudhead): Fetch branch from backend.
       commit.branch = "master";
+      commit.changeset = {
```

Comment:
You could move this mock out into a separate file (maybe `commitMocks.js`?), we already have one like that: https://github.com/radicle-dev/radicle-upstream/blob/master/ui/lib/orgMocks.js

Reply:
I see @xla moved it into the store, that works for me as well.